featurebyte/feature_manager/manager.py

```diff
@@ -73,7 +77,10 @@ def __init__(
         )
 
     async def online_enable(
-        self, feature_spec: OnlineFeatureSpec, schedule_time: datetime = datetime.utcnow()
+        self,
+        feature_spec: OnlineFeatureSpec,
+        schedule_time: datetime = datetime.utcnow(),
```

Comment:
Maybe we should use `Optional[datetime] = None` here and set the `schedule_time = datetime.utcnow()` if it is not set. Otherwise, the `schedule_time` value will be equal to a constant value.

Reply:
Indeed, looks like a bug. Will fix it in this PR. FYI @yanhuisg